services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusServiceITest.kt

```diff
@@ -817,6 +825,28 @@ class HakemusServiceITest(
             }
         }
 
+        @Test
+        fun `saves the Allu ID and status even when sending the form data attachment fails`() {
+            val hakemus = hakemusFactory.builder().withMandatoryFields().save()
+            every { cableReportService.create(any()) } returns alluId
+            every { cableReportService.addAttachment(alluId, any()) } throws AlluException(listOf())
```

Comment:
Maybe `AlluException` should have a default (empty) value for the list to make this just `throws AlluException()`

Reply:
AlluException is only used in tests, so I moved it to test code. The parameter was never used, so I removed it.